Sources/FluentUI_macOS/Components/DatePicker/CalendarHeaderView.swift

```diff
@@ -28,6 +28,66 @@ private struct Constants {
 	private init() {}
 }
 
+/// Allows the top row of the `CalendarHeaderView` to function as an accessible
+/// stepper for moving between months. Increment to move forwards, decrement to
+/// move backwards.
+class AccessibleCalendarHeaderStackView: NSStackView, NSAccessibilityStepper {
+	init(monthYearLabel: NSTextField, leadingButton: NSButton, trailingButton: NSButton) {
+		self.monthYearLabel = monthYearLabel
+		self.leadingButton = leadingButton
+		self.trailingButton = trailingButton
+
+		super.init(frame: .zero)
+
+		self.setAccessibilityElement(true)
+		self.setAccessibilityRole(.incrementor)
+
+		self.addView(monthYearLabel, in: .center)
+		self.addView(leadingButton, in: .leading)
```

Comment:
Should we mark these nested controls as not accessibility elements to match OS pattern of stepper elements not having children?